packages/wasm-sdk/check_documentation.py

```
@@ -72,15 +68,11 @@ def check_documentation_completeness():
     
     for cat_key, category in current_defs.get('queries', {}).items():
```

Ruff (0.12.2)

Loop control variable `cat_key` not used within loop body

Rename unused `cat_key` to `_cat_key`

(B007)